MobiFlight/MobiFlightLedModule.cs

```diff
@@ -65,6 +66,14 @@ public void Display(int subModule, String value, byte points, byte mask)
 
             var command = new SendCommand((int)MobiFlightModule.Command.SetModule);
 
+            var strValueParts = value.Split('.');
+            value = String.Join("", strValueParts);
+
+            if (strValueParts.Length > 1)
+            {
+                points = CalculateCorrectPoints(String.Join(".", strValueParts), mask);
```

Comment:
Then here instead of having to re-join you can just pass in the original `value`.